platforms/intel-iot/edison/sparkfun_pinmap.go

```diff
@@ -0,0 +1,116 @@
+package edison
+
+var sparkfunPinMap = map[string]sysfsPin{
+  "12": sysfsPin{
```

Comment:
I did think about that as it is what is on the board it self.  However most of the documentation including a lot of the Intel stuff references just the port number eg `echo 14 > /sys/class/gpio/export`.  When I came to try the code for the first time my natural instinct was to use "14" so thats what swayed it in my mind.....